packages/e2e-tests/helpers/test-app.ts

```diff
@@ -63,7 +64,7 @@ export default class TestApp {
     ])
 
     await chooser.setFiles(filepaths)
-    await this.mainWin.locator("text=Import Complete.").isVisible()
```

Comment:
Looks like this might be better. You won't need expect.

```
await this.mainWin.getByText("Load Successful").waitFor()
```

https://playwright.dev/docs/api/class-locator#locator-wait-for

Reply:
@jameskerr: Thanks for finding that! Indeed, I tried it out and it worked as advertised. I've pushed a commit to use that instead and backed out the linter changes I'd put in to permit expect.